### PR TITLE
Fixes and adds some sounds

### PR DESCRIPTION
**Pull request by MrAsbestos**
### Purpose
This pull request adds and fixes some sounds. (duh)

### Changelog:
-Fixed carps and xenos not having attack sounds.
-Partially fixed window breaking sounds not playing.
-Modfied the Buzzsaw firing sounds (now it is more impactful).
-Fixed all of the foam guns not having firing sounds.
-Added new captain and centcom announcements (will try to implement them later).

### CODE REVIEW

**Comment by MrAsbestos** [comment]
Ready to merge btw

File: UnityProject/AddressablePackingProjects/SoundAndMusic/Assets/AddressableAssetsData/AssetGroups/Music.asset [hunk 1]
```diff
@@ -239,6 +239,18 @@ MonoBehaviour:
     m_SerializedLabels: []
     m_MainAsset: {fileID: 0}
     m_TargetAsset: {fileID: 0}
+  - m_GUID: fab582363ff3d7c478a69bc0da678df2
+    m_Address: Assets/Prefabs/AI/captain_announcement.prefab
```
Comment (corp-0) [inline UnityProject/AddressablePackingProjects/SoundAndMusic/Assets/AddressableAssetsData/AssetGroups/Music.asset:243]: I don't think these are music sounds, are they?

Comment (MrAsbestos) [reply UnityProject/AddressablePackingProjects/SoundAndMusic/Assets/AddressableAssetsData/AssetGroups/Music.asset:243]: huh that's weird
I never touched music

**Comment by corp-0** [comment]
if I remember correctly, the announce sounds are part of the common sounds SO so you should be able to simply change them there to *implement* the new sounds.

**Comment by MrAsbestos** [comment]
Yeah I tried that, you know what I am gonna try again, I think I know what I did wrong

**Comment by MrAsbestos** [comment]
Alright! I implemented the new captains announcement sound, will commit later
Didn't find the one for centcom tho

**Comment by MrAsbestos** [comment]
huh this is weird
I changed the sounds but it doesn't show up on my git, so I can't commit

**Comment by MrAsbestos** [comment]
Ready to merge ;)

**Pull request merged**